src/components/QuestionnaireData.vue

```diff
@@ -1,9 +1,80 @@
 <template>
-  <div>
-    <h2>WORLD!!</h2>
-     <div id="nav">
-      <router-link to="/">SPI</router-link>
+  <div id="questionnaire">
+    <div v-bind:class="container">
+      <div v-bind:class="titleBox">
+        <p v-bind:class="stepBox">STEP2</p>
+        <p v-bind:class="titleText">以下にお答えください</p>
+      </div>
+      <div v-bind:class="questionnaireBox">
+        <form>
+          <div>
+            <p v-bind:class="spi">現在、生命保険に加入されていますか？</p>
+            <input type="radio" name="insurance" v-model="insurance" v-on:click="changeHospiShow" value="はい">はい
+            <input type="radio" name="insurance" v-model="insurance" v-on:click="changeHospiShow" value="いいえ">いいえ
+          </div>
+
+          <div v-if="hospiShow">
+            <p v-bind:class="spi">現在入院中ですか。または、直近3か月以内に医師の診察・検査の結果、入院・手術をすすめられたことはありますか？</p>
+            <input type="radio" name="hospitalization"  v-model="hospitalization" v-on:click = "changeSurgey" value="はい">はい
+            <input type="radio" name="hospitalization" v-model="hospitalization" v-on:click = "changeSurgey = true" value="いいえ">いいえ
+          </div>
+
+          <div v-if="surgeyShow">
+            <p v-bind:class="spi">過去5年以内に病気やケガで手術をうけたこと、または継続して7日以上の入院をしたことがありますか？</p>
+            <input type="radio" name="surgery" v-model="surgery" value="はい">はい
+            <input type="radio" name="surgery" v-model="surgery" value="いいえ">いいえ
+          </div>
+        </form>
+      </div>
+    </div>
+    <div v-bind:class="btnBox">
+      <button v-bind:class="backBtn"><router-link to="/">前へ戻る</router-link></button>
+      <router-view />
+      <button v-bind:class="nextBtn"><router-link to="/consultation">次へ進む</router-link></button>
+      <router-view />
     </div>
-    <router-view />
   </div>
-</template>
\ No newline at end of file
+</template>
+
+<script>
+  export default {
+    data: function() {
+      return {
+        /*モデル*/
+        insurance: '',
+        hospitalization: '',
+        surgery: '',
+        /*真偽値*/
+        hospiShow: false,
+        surgeyShow: false,
+        /*CSSクラス*/
+        container: 'container',
+        titleBox: 'title-box',
+        stepBox: 'step-box',
+        titleText: 'title-text',
+        questionnaireBox: 'questionnaire-box',
+        spi: 'spi',
+        btnBox: 'btn-box',
+        nextBtn: 'next-btn',
+        backBtn: 'back-btn'
+      }
+    },
+    methods: {
+      changeHospiShow: function() {

```

Comment:
以下のような書き方もあるので、ぜひ使ってみて下さい。
```suggestion
      changeHospiShow() {

```